src/app/public/plugin-resources/code-examples/character-count/character-count-demo.component.html

```diff
@@ -0,0 +1,43 @@
+<div
+  style="width: 50%;"
```

Comment:
Do we need this div here for the code example? I know it is just the demo code but there isn't truly a reason for it to be here for the example unless I'm missing something.

Reply:
I just based this on the design spec (see issue for link).

Reply:
Right. I get that for the demo - just didn't know if it needed to be there for the code example. Not a big deal either way.